actk/bin/all.py

```diff
@@ -82,24 +84,32 @@ def run(
                 # Create or get log dir
                 # Do not include ms
                 log_dir_name = datetime.now().isoformat().split(".")[0]
-                log_dir = Path(f"~/.dask_logs/actk/{log_dir_name}").expanduser()
+                log_dir = Path(f".dask_logs/{log_dir_name}").expanduser()
                 # Log dir settings
                 log_dir.mkdir(parents=True, exist_ok=True)
 
+                # Configure dask config
+                dask.config.set(
+                    {
+                        "scheduler.work-stealing": False,
+                        "logging.distributed.worker": "info",
+                    }
+                )
+
                 # Create cluster
                 log.info("Creating SLURMCluster")
                 cluster = SLURMCluster(
-                    cores=1,
-                    memory="4GB",
+                    cores=4,
```

Comment:
Would it be worth making these environment variables with defaults so they can be tweaked without having to edit this file?